### ISSUE TITLE
Doesn't respond to current directory changes

### ISSUE THREAD

**Issue by kalafut** [romshark/templier#7]
Hi. Good package, and things mostly work well, but I've noticed that the rebuilding and/or restarts do not occur if the files changed are in the current directory (i.e. from which templier was run). I have my config referencing `./`, but templier doesn't react to any edits to files in `./`. This includes the example config change custom builder... not just Go files. If I move things into a subdirectory, then changes to them are picked up.

**Comment by kalafut** [comment]
I did a little debugging which led me to the `add()` function: https://github.com/romshark/templier/blob/main/internal/watcher/watcher.go#L279

It doesn't watch `dir` that is passed in... only the child directories.

As a hacky experiment I added this to the top of the function:

```go
	d, err := filepath.Abs(dir)
	if err != nil {
		return err
	}
	if err := w.watcher.Add(d); err != nil {
		return err
	}
	w.watchedDirs[d] = struct{}{}
```

Then I saw what I think is correct behavior, with edits to root directory files triggering rebuilds, etc. But there is quite a bit going on in the tool, so I may not be understand this all correctly.

**Comment by kalafut** [comment]
I think I found the issue. I used the [example config](https://github.com/romshark/templier/blob/main/example-config.yml#L46) to get started, and it has the line:

```yaml
  exclude:
    - .* # all hidden files and directories
```

The issue is that when processing the directories, the relative path for the root directory is ".", which matches `.*` as an excluded directory. When I changed the exclusion to `.?*` (mandate at least one character after the dot), things seem to work OK. It may be worth updating the example config if the code is otherwise correct.

**Comment by romshark** [comment]
Thanks for going the extra steps debugging the issue!
There indeed was a bug and Templier shouldn't have excluded the working directory based on `".*"`.
This is fixed now https://github.com/romshark/templier/commit/86d16df680a9c80f383ff6b9d1772cbc5ef057cb and released under [v0.7.1](https://github.com/romshark/templier/releases/tag/v0.7.1)
